python/src/aiconfig/eval/lib.py

```diff
@@ -162,7 +162,7 @@ def _extract_value(input_text_datum: TextBasedInputDatum | None) -> str | None:
                     case str(input_text):
                         return input_text
                     case frozendict():
-                        return json.dumps(input_text_datum.value)
+                        return json.dumps(input_text_datum.value, sort_keys=True)
```

Comment:
Nit: needed for serialization of dict --> text for testing, and also have it be deterministitc in canonical form